xds/build.gradle

```diff
@@ -53,6 +53,7 @@ configureProtoCompilation()
 jar {
     classifier = 'original'
 }
+configurations.archives.artifacts.removeAll { it.classifier == "original" }
```

Comment:
This doesn't look right. With maven-publish artifacts should no longer really do anything/we shouldn't be using. I would have expected this in the `publishing` section, like seen in alts.

Reply:
Oops, I was exactly copying alts/build.gradle, but the version of my branch was earlier than 1c3432c3fb055e40beeb8aee38e114834ca10da5